eva_sub_cli_processing/process_jobs.py

```diff
@@ -53,47 +30,105 @@ def _process_submission(submission):
 class SubmissionScanner(AppLogger):
 
     statuses = []
+    step_statuses = []
 
-    def scan(self):
+    def _scan_per_status(self):
```

Comment:
I was initially very confused about why we needed to scan both tables, before I realised that this scan is (I think) only used to add the first processing step. If that's the case, then maybe it could be a bit less generic and used only in that specific situation - even if we used it for other operations (e.g. scanning for cancelled submissions to clean up the db or something), I don't think creating a `SubmissionStep` for validation would make sense in those cases.